src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -106,6 +117,10 @@ export const WorkspaceList = () => {
       name: 'Features',
       isExpander: true,
       hasActions: true,
+      render: (features: string[]) => {
+        const { total, selected } = getSelectedFeatureQuantities(features, configurableApps || []);
```

Comment:
Shall we render "-" for configurableApps is undefined or empty? `0/0` may looks strange.

Reply:
I don't think so.  Usually `-` represents null or missed data, `0/0` represent 0 features selected, I think that's how it should be.